.circleci/config.yml

```diff
@@ -7,7 +7,7 @@ executors:
   ruby-latest:
     resource_class: small
     docker:
-      - image: circleci/ruby:latest
+      - image: circleci/ruby:2.6.6
```

Comment:
Required to make gem dependencies happy.  The alternative is to force local development on ruby 3.